shared/js/media/media_frame.js

```diff
@@ -371,6 +393,11 @@ MediaFrame.prototype._displayImage = function(backgroundImage, width, height) {
   var temp = this.image.clientLeft; // jshint ignore:line
 };
 
+MediaFrame.prototype.localize = function localize() {
+  if (this._localize) {
```

Comment:
Please just put the accessibliity code directly here. You may need to use this.displayingImage or this.displayingVideo to distinguish the two cases.  And maybe add a comment that this localization code is really just localizing the accessability information.